src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
+    Orange,
+    Yellow,
+    Green,
```

Comment:
```suggestion
    RED,
    ORANGE,
    YELLOW,
    GREEN,
```

we need to use CAPS for Enums elements